src/dawn/engine_dawn.cc

```diff
@@ -502,18 +520,41 @@ Result EngineDawn::CreateFramebufferIfNeeded(
   if (!result.IsSuccess())
     return result;
 
-  // First make the Dawn color attachment textures that the render pipeline will
-  // write into.
   {
     ::dawn::Texture fb_texture;
 
-    result =
-        MakeFramebufferTexture(*device_, fb_format, width, height, &fb_texture);
+    result = MakeTexture(*device_, fb_format, width, height, &fb_texture);
     if (!result.IsSuccess())
       return result;
     render_pipeline->fb_texture = std::move(fb_texture);
   }
 
+  // Then make the depthStencil attachment textures that the render pipeline
```

Comment:
We have a choice here:

1.  Always create a (Dawn) depth-stencil texture and attach it to the pipeline.  The existence of the depth-stencil buffer on the ::amber::Pipeline only controls the format.

2.  Only create the Dawn depth-stencil texture if the Amber depth-stencil texture exists.  

Option 1 is simpler from a flow perspective because the depth-stencil codepath is always exercised.  But it takes more resources on the GPU.  Also, it allocates a texture in the Dawn engine when the Amber side of the engine doesn't have one.  

On balance I think option 2 is better.  It also means you don't need to add the default depth-stencil format constant kDepthStencilFormat.  (And that's how I actually noticed this design issue.)

Reply:
Yes, option 2 makes more sense.